week2/homework/src/index.js

```diff
@@ -1,3 +1,119 @@
-'use strict';
-
-// TODO: Write the homework code in this file
+'use strict';
+
+const fs = require('fs');
+const storageFile = 'todolist.json';
+
+function readFile() {
+  fs.readFile(storageFile, 'utf8', (err, list) => {
+    if (err) {
+      throw err;

```

Comment:
who will catch this error? Perhaps it is enough to tell that user that something went wrong `if ( err) { console.error('Ooops'); return; }`